src/plug/Views/TimeLimitView.vala

```diff
@@ -135,16 +135,24 @@ namespace PC.Widgets {
             enable_switch.halign = Gtk.Align.START;
             enable_switch.valign = Gtk.Align.CENTER;
 
+            var today_local = new GLib.DateTime.now_local ();
+            var today_start = new GLib.DateTime.local (today_local.get_year (), today_local.get_month (),
+                                                      today_local.get_day_of_month (), 0, 0, 0);
+            var today_end = new GLib.DateTime.local (today_local.get_year (), today_local.get_month (),
+                                                    today_local.get_day_of_month (), 23, 59, 59);
```

Comment:
Indentation nit pick, otherwise this is great!

```suggestion
            var today_start = new GLib.DateTime.local (
                today_local.get_year (),
                today_local.get_month (),
                today_local.get_day_of_month (),
                0, 0, 0
            );

            var today_end = new GLib.DateTime.local (
                today_local.get_year (),
                today_local.get_month (),
                today_local.get_day_of_month (),
                23, 59, 59
            );
```